src/RMP/Translate/lang/programmes/cy.po

```diff
@@ -1049,6 +1049,10 @@ msgstr ""
 msgid "latest"
 msgstr "Diweddaraf"
 
+#. English: "Latest episode".
+msgid "latest_episode"
+msgstr "Pennod Ddiweddaraf"
```

Comment:
I was going by designs. Ran it by a designer and they said yes it should be lower case.
Sorry.